src/main/java/org/gatorgradle/display/CommandOutputSummary.java

```diff
@@ -114,15 +114,22 @@ public void uploadOutputSummary(List<CheckResult> failed, List<CheckResult> all)
     StringBuilder builder = new StringBuilder();
     builder.append("{");
 
-    String userEmail = "unknown";
-    BasicCommand getGitUser = new BasicCommand("git", "config", "--global", "user.email");
-    getGitUser.run();
-    if (getGitUser.exitValue() == Command.SUCCESS) {
-      userEmail = getGitUser.getOutput().trim();
+    String userInfo = "unknown";
+    String userAttribute = "unknown";
+    if (GatorGradleConfig.get().shouldUsernameEnvVar() == true) {
+      userInfo = System.getenv("TRAVIS_REPO_SLUG");
+      userAttribute = "repo";
+    } else {
+      BasicCommand getGitUser = new BasicCommand("git", "config", "--global", "user.email");
+      getGitUser.run();
+      if (getGitUser.exitValue() == Command.SUCCESS) {
+        userInfo = getGitUser.getOutput().trim();
+      }
+      userAttribute = "user";
```

Comment:
I think this is a bit too complicated, and would make processing this information more difficult; instead of using 'userAttribute`, just put whatever value you get, regardless of where it comes from, in an `id` attribute.

Reply:
On the other hand, I can see how identifying this would be useful...